### PR TITLE
Add block_gossip SSE event

### PR DESCRIPTION
**Pull request by dapplion**
Complements https://github.com/ethereum/beacon-APIs/pull/404 to differentiate two stages of block validity.

- `block`: block imported on the fork-choice `on_block` handler
- `block_gossip`: passes validation rules of the `beacon_block` topic

See discussion and motivation in https://github.com/ethereum/beacon-APIs/issues/349

### CODE REVIEW

**Review by g11tech — APPROVED** [review]
lgtm

**Review by rolfyone — DISMISSED** [review]
should add to changelog but looks ok to me other than that.

File: apis/eventstream/index.yaml [hunk 1]
```diff
@@ -55,6 +56,11 @@ get:
               value: |
                 event: block
                 data: {"slot":"10", "block":"0x9a2fefd2fdb57f74993c7780ea5b9030d2897b615b89f808011ca5aebed54eaf", "execution_optimistic": false}
+            block_gossip:
+              description: The node has received a block (from P2P or API) that passes validation rules of the `beacon_block` topic
+              value: |
+                event: block
```
Comment (rolfyone) [inline apis/eventstream/index.yaml:62]: actually i think this name needs to be `block_gossip`

**Comment by rolfyone** [comment]
@dapplion just changelog now :)

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**